tests/tests_suite_test.go

```diff
@@ -89,6 +90,19 @@ func (s *newSspStrategy) Init() {
 	Eventually(func() error {
 		return apiClient.Create(ctx, newSsp)
 	}, timeout, time.Second).ShouldNot(HaveOccurred())
+
+	Eventually(func() error {
+		return apiClient.Create(ctx, getTestTemplate("3", strategy.GetTemplatesNamespace(), commonTemplates.Version))
```

Comment:
Ah, I think that answers my previous comment and also explains the failed default-variant test mentioned above.

Reply:
Why do we need to create these templates at test suite init? If they are only relevant for a specific test, they should be created and cleaned up in the context of the test